test/test_v2_dataset_warnings.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+
+def test_warns_if_imported_from_datasets():
+    import torchvision
+
+    value = torchvision._WARN_ABOUT_BETA_TRANSFORMS
+
+    setattr(torchvision, "_WARN_ABOUT_BETA_TRANSFORMS", True)
```

Comment:
You can do that with `mocker.patch_object`: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch.object

This way you don't have to reset in manually.